src/crab/type_to_num.cpp

```
@@ -204,12 +242,19 @@ void TypeToNumDomain::havoc_all_locations_having_type(const TypeEncoding type) {
             values.havoc(variable_registry.kind_var(kind, type_variable));
         }
     }
+    // Drop every binding in the region just havoced.
+    if (!ptr_sum_bindings.empty()) {
+        std::erase_if(ptr_sum_bindings,
```

Cppcheck (2.21.0)

[performance] Function parameter 'premise_set' should be passed by const reference.

(passedByValue)